resources/jest-preprocessor.js

```diff
@@ -1,12 +1,17 @@
+/* tslint:disable */
+
 const tsc = require('typescript')
 const tsConfig = require('../tsconfig.json')
 
-const { compilerOptions } = tsConfig
+const compilerOptions = tsConfig.compilerOptions
 
 module.exports = {
   process (src, fileName) {
     // Make sure that source maps are always enabled and inline so that they
     // can be picked up by the `source-map-support` package.
-    return tsc.transpileModule(src, { compilerOptions, fileName }).outputText
+    return tsc.transpileModule(src, {
+      compilerOptions: compilerOptions,
+      fileName: fileName,
+    }).outputText
```

Comment:
I'm assuming these changes are just necessary to run this file direct on Node 4 since it doesn't support destructuring and object literal shorthand syntax.